### PR TITLE
#174797529 Users/Managers should be able to comment on travel req…

### PR DESCRIPTION
**Pull request by BeyFaith**
#### What does this PR do?
Users/Managers are able to comment on travel requests.
#### Description of Task to be completed?
- Pull the repo to get the project's latest version
- Create a branch to work on
- Select comments from the database.
- Create a comment 
- Validate comments 
- Write Tests
- Add, commit, and push changes to GitHub
#### How should this be manually tested?
Clone this repo, run npm run dev to run the project. After having a project running on a port using localhost:PORTNO to run the project in the browser. Go to http://localhost:PORTNO/api/comments . After sending the request, the message of success and the comment will be returned.
#### What are the relevant pivotal tracker stories?
[#174797529](https://www.pivotaltracker.com/story/show/174797529)

### CODE REVIEW

**Review by dushimeemma — APPROVED** [review]
comment model, comment migration need new line at the end and english locales need to have allowed spacing and new line at the end also work on language translation...otherwise smells good to me

**Comment by valens10** [comment]
Screenshot from 2020-11-03 15-49-47
This is what I am getting

**Comment by valens10** [comment]
Screenshot from 2020-11-03 15-49-47
This is what i am getting you also need to rebase.

File: locales/fr.json [hunk 1]
```diff
@@ -164,9 +164,21 @@
...
+  "Successful": "Réussie",
+  "Comment does not exist": "Le commentaire n'existe pas",
+  "Success": "Succès",
+  "Successfully deleted a comment": "Un commentaire a bien été supprimé",
+  "Successfully updated a comment": "Mise à jour réussie d'un commentaire",
+  "Comment should be string": "Le commentaire doit être une chaîne",
+  "Comment input can't be empty": "L'entrée du commentaire ne peut pas être vide",
+  "Comment is required": "Un commentaire est requis"
+>>>>>>> feat(comments):Users/Managers should be able to comment on travel requests
 }
```
Comment (elvisiraguha) [inline locales/fr.json]: Fix these

**Review by elvisiraguha — APPROVED** [review]

**Review by b0nbon1 — APPROVED** [review]

**Pull request merged**